Userland/Applications/Spreadsheet/SpreadsheetView.cpp

```diff
@@ -204,6 +204,15 @@ void InfinitelyScrollableTableView::mouseup_event(GUI::MouseEvent& event)
     }
 }
 
+void InfinitelyScrollableTableView::drop_event(GUI::DropEvent& event)
```

Comment:
Again, please write commit messages with more care, and especially make sure to follow the code submission policy.